core/templates/domain/exploration/StatesObjectFactory.ts

```diff
@@ -47,6 +47,8 @@ export interface WrittenTranslationObjectsDict {
   [stateName: string]: WrittenTranslation[];
 }
 
+type InteractionSpecsKey = keyof typeof INTERACTION_SPECS;
```

Comment:
Didn't you already define this elsewhere?

Reply:
Yeah exporting it from here and using elsewhere now